Units/parser-eiffel.r/obsolete.e.d/input.e

```diff
@@ -1,4 +1,4 @@
-class OBSOLETE obsolete "Use a different one"
+class OBSOLETE_ obsolete "Use a different one"
```

Comment:
I see. I read change logs of exuberant ctags. However, I cannot find any information about this.
Your mention looks reasonable for me.